rocketc/rocketc.py

```diff
@@ -257,8 +257,9 @@ def init(self):
         if response['success']:
             response = response['data']
             user_id = response['userId']
+            auth_token = response['authToken']
 
-            self._join_user_to_groups(user_id, user_data)
+            self._join_user_to_groups(user_id, user_data, auth_token)
```

Comment:
why the auth_token is required now? this question applies for all the methods requiring this new param

Reply:
because I have to get all the groups where the user has presence and the API lists all of the private groups the calling user has joined, so the request must be done for the currently user not for the admin, then this need the auth token and user id. 
https://rocket.chat/docs/developer-guides/rest-api/groups/list/

Another options is get all the private groups and then filter by user, but this could need more time to complete the task